app/public/lobby/lobbyfuncs.js

```diff
@@ -39,68 +44,48 @@ const addRowToTopOfTable = (table, values) => {
 	lobbyTbl.prepend(gameRow);
 }
 
-createBtn.addEventListener("click", () => {
-	if (joinedLobbyId === 0) {
-		getValidId();
-		post(
-			"/lobby/make",
-			{"lobbyId" : lobbyId, "username" : username}
-		).then((resp) => {
-			if (resp.status !== 200) {
-				// TO DO: Handle errors in joining lobby when lobby system is done
-				console.log("ERROR - Something went wrong");
-			}
-			else {
-				joinedLobbyId = lobbyId;
-				lobbyId++;
-			}
-			// table should update shortly
-		}).catch((e) => {
-			console.log(e);
-		});
-	}
-	else {
-		console.log("stop spamming!");
-		// to avoid weird effects when one client makes a bunch of lobbies
-		// temporary
-	}
-	
-});
-
-// lobby refreshes automatically, so this is only needed if a lobby is removed
-// any other buttons we should have?
-refreshBtn.addEventListener("click", () => {
-	fetch(
-		"/lobby/list"
-	).then((resp) => {
-		if (resp.status !== 200) {
-			// TO DO: Handle errors in joining lobby when lobby system is done
-			console.log("ERROR - Something went wrong");
-		}
-		// update table according to list
-		fetchRefresh();
-		
-	}).catch((e) => {
-		console.log(e);
-	});
-	
-});
-
-
 WS.onConnect(() => {
 	fetch(
 		"/lobby/list"
 	).then((resp) => {
 		if (resp.status !== 200) {
-			// TO DO: Handle errors in joining lobby when lobby system is done
 			console.log("ERROR - Something went wrong");
 		}
 		// update table according to list
-		fetchRefresh();
-		
+		resp.json().then((lobbyList) => {
+			console.log(lobbyList);
+			refreshLobbies(lobbyList);
+			
+			post(
+				"/lobby/joinstatus",
+				{"username" : username}
+			).then((resp) => {
+				if (resp.status !== 200) {
+					console.log("ERROR - Something went wrong");
+				}
+				// update page according to data
+				resp.json().then((data) => {
+					// if refresh while in lobby, user goes back to lobby
+					console.log(data);
+					if (data.lobbyId !== undefined) {
+						joinedLobbyId = data.lobbyId;
+						showJoinedLobbyUI();
```

Comment:
This isn't working for me. When I refresh, I am no longer in the lobby. If I click the row, the game begins.
Is this because the username system isn't working yet?